libbeat/monitoring/monitoring.go

```diff
@@ -4,6 +4,7 @@ import "errors"
 
 type Mode uint8
 
+//go:generate stringer -type=Mode
```

Comment:
oh, it's a leftover from debugging. There are 2 problems with stringer though:
1. you either need to have it installed or we have to vendor it and use `go run`
2. package loading in stringer requires you to have done `go install` on dependent packages (unless you have build a binary already). Otherwise `go generate` will fail (See [this ticket](https://github.com/golang/go/issues/10249)).